static/js/components/DBStats.jsx

```diff
@@ -41,9 +42,25 @@ const DBStats = () => {
                 <em>{key}</em>
               </TableCell>
               <TableCell>
-                {!Number.isNaN(Number(dbStats[key]))
-                  ? Number(dbStats[key]).toLocaleString()
-                  : dbStats[key]}
+                {/* eslint-disable-next-line no-nested-ternary */}
```

Comment:
Nested ternaries makes for difficult to parse code; perhaps best to heed this warning from eslint.

Reply:
Done